src/main/java/seedu/address/ui/CompletableDeadlineCard.java

```diff
@@ -27,6 +27,18 @@
     @FXML
     private Label completedLabel;
 
+    /**
+     * Creates a {@code CompletableDeadlineCard} with the given {@code CompletableDeadline} without an index to display.
+     */
+    public CompletableDeadlineCard(CompletableDeadline deadline) {
+        super(FXML);
+        this.deadline = deadline;
+        id.setText("");
+        description.setText(deadline.getDescription());
+        date.setText(DateUtil.decodeDate(deadline.getBy()));
+        completedLabel.setText(deadline.getIsDone() ? "✔" : "");
```

Comment:
I remember the implementation here was changed, or is this correct?

Reply:
Card to display UI elements (either tick to empty), deadline class will not do UI related stuff.